### PR TITLE
Update integration tests to use Ficus

### PR DESCRIPTION
**Pull request by haikuginger**
This pull request updates our integration tests to use Ficus, rather than Eucalyptus, which no longer works due to missing dependencies.

### CODE REVIEW

**Comment by haikuginger** [comment]
@smarnach, tests are passing!

File: playbooks/collect_activity/collect_activity.yml [hunk 1]
```diff
@@ -15,13 +15,13 @@
       copy:
         src: stats.py
         dest: /tmp/collect_activity.py
-      become_user: www-data
+      become: yes
```
Comment (smarnach) [inline playbooks/collect_activity/collect_activity.yml]: So you are running the collect_activity script as root now?  Why is this necessary?

@bdero Would you have any concerns about this change?

Comment (haikuginger) [inline playbooks/collect_activity/collect_activity.yml]: It looks like www-data doesn't have even read access for any of the requisite files anymore, and I figured that, since this is just a test, it was a better choice to just use a user that'll always have the necessary privileges than to spend time figuring out who has those privileges now, to be further updated as needed.

Comment (bdero) [inline playbooks/collect_activity/collect_activity.yml]: @smarnach Yes, we should definitely try to keep these scripts running with the same user as the LMS/CMS uses at runtime.

`collect_activity.py` initializes Django in the LMS/CMS environment, and my main concern is that doing this as root may cause subtle issues down the line that are difficult to track down if, for example, the Django setup process caches anything to disk (which I suspect does happen, including with python bytecode caching, etc.), or if edX decides to add anything to the setup process that could result in files being written.

Comment (haikuginger) [inline playbooks/collect_activity/collect_activity.yml]: All right; let me see if this works when run as `edxapp`. :)

**Comment by haikuginger** [comment]
After some digging, located this thread:

https://github.com/ansible/ansible/issues/14635

It looks like `become_user` does not automatically set `become: yes`. So, the fact that testing this command worked previously was likely a total fluke; our default SSH user used to have access to the directories in question, but no longer does. Adding `become: yes` to the existing `become_user: www-data` should fix it.

File: playbooks/collect_activity/collect_activity.yml (hunk 1 above)
Comment (smarnach) [inline playbooks/collect_activity/collect_activity.yml]: @haikuginger Re "since this is just a test": it isn't just a test.  The playbook is used for production instances; otherwise I wouldn't be that worried.

**Comment by smarnach** [comment]
@haikuginger Thanks for figuring this out!  The changes look good to me, :+1:.  I'm merging and deploying this now, so I can rebase #198 and get its build passing.

**Comment by bdero** [comment]
@haikuginger Ah, thanks for tracking that down!
